### PR TITLE
DB-Zoos - Carlos Vargas

### PR DESCRIPTION
**Pull request by cornielleandres**

### CODE REVIEW

File: data/helpers/bearDb.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const db = require('../dbConfig.js');
```
Comment (zackhitch) [inline data/helpers/bearDb.js]: Very well written. 🔥

File: data/helpers/zooDb.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const db = require('../dbConfig.js');
```
Comment (zackhitch) [inline data/helpers/zooDb.js]: Well done!

File: routes/bears/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		.catch(err => res.status(500).json({ error: `Bear information could not be retrieved: ${ err }` }));
+});
+
+// get bears with given ID
+router.get('/:id', (req, res) => {
+	const { id } = req.params;
+	bearDb
+		.get(id)
+		.then(bear => {
+			if (bear.length) return res.status(200).json(bear[0]);
```
Comment (zackhitch) [inline routes/bears/index.js]: Great error handling

File: routes/bears/index.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		})
+		.catch(err => res.status(500).json({ error: `There was an error while getting the information for bear with ID ${ id }: ${ err }` }));
+});
+
+// post new bear and return ID of newly created bear
+router.post('/', (req, res) => {
+	const name = req.body;
+	bearDb
+		.insert(name)
+		.then(id => res.status(201).json(id.id[0]))
```
Comment (zackhitch) [inline routes/bears/index.js]: Think about adding some error handling here.

File: routes/bears/index.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+});
+
+// update bear with given ID
+router.put('/:id', (req, res) => {
+	const { id } = req.params;
+	const name = req.body;
+	bearDb
+		.update(id, name)
+		.then(putBool => {
+			if (putBool) return res.status(200).json(`Bear with ID ${ id } was updated successfully`);
```
Comment (zackhitch) [inline routes/bears/index.js]: Very nice error handling! 🙌

File: routes/zoos/index.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+// get zoo with given ID
+router.get('/:id', (req, res) => {
+	const { id } = req.params;
+	zooDb
+		.get(id)
+		.then(zoo => {
+			if (zoo.length) return res.status(200).json(zoo[0]);
+			return res.status(404).json({ error: `Zoo with ID ${ id } does not exist` });
+		})
+		.catch(err => res.status(500).json({ error: `There was an error while getting the information for zoo with ID ${ id }: ${ err }` }));
```
Comment (zackhitch) [inline routes/zoos/index.js]: While one-liners are fun to write, they sometimes are not as easy to read as you'd think. Some of your one-liners are too long to be easily read. Line breaks and keeping line length shorter, makes your code easier to read than many one-liners.

Comment (cornielleandres) [reply routes/zoos/index.js]: I've started implementing your observations into my code starting today. Thanks.

**Pull request closed without merging**